composer/trainer/trainer.py

```diff
@@ -920,24 +916,25 @@ def __init__(
         # Load Checkpoint
         self._rng_state = None
         # If autoresume is enabled, first check for existing checkpoints to load
-        latest_checkpoint_path = self._check_for_autoresume(autoresume=autoresume,
-                                                            save_folder=save_folder,
-                                                            save_overwrite=save_overwrite,
-                                                            save_latest_filename=save_latest_filename,
-                                                            run_name=run_name,
-                                                            save_latest_artifact_name=save_latest_artifact_name,
-                                                            loggers=loggers,
-                                                            load_chunk_size=load_chunk_size,
-                                                            load_progress_bar=load_progress_bar)
-        # Found latest checkpoint path, load that instead
-        if latest_checkpoint_path:
-            load_path = latest_checkpoint_path
-            # Disable object_store and load_weights_only since we're autoresuming locally
-            load_object_store = None
-            load_weights_only = False
-            log.info(
-                f"Found latest checkpoint at {latest_checkpoint_path}, loading instead of load_path {load_path} as autoresume enabled."
-            )
+        if autoresume:
```

Comment:
related to autoresume not docs

Reply:
You're right, I shouldn't have done this in this PR.

I made this change as we've been going with the convention that the caller should do the `if` check (i.e. `if x: do_y()`) rather than having the callee handle potentially-falsy values (i.e. `def y(x): return if not x else ...`), and figured it was easier to refactor it and then write a simpler docstring.